mobx/lib/model/todo_list.dart

```diff
@@ -6,6 +6,8 @@ part 'todo_list.g.dart';
 
 class TodoList = _TodoList with _$TodoList;
 
+enum AppTab { todos, stats }
```

Comment:
How about the name `TabType` ?

Reply:
Yea that makes more sense. I'll make the change.